drv/i2c-devices/src/mwocp68.rs

```diff
@@ -350,6 +440,262 @@ impl Mwocp68 {
         Ok(val)
     }
 
+    /// Will return true if the device is present and valid -- false otherwise
+    pub fn present(&self) -> bool {
+        match Mwocp68::validate(&self.device) {
+            Ok(valid) => valid,
+            _ => false,
+        }
+    }
+
+    pub fn power_good(&self) -> Result<bool, Error> {
+        use commands::mwocp68::STATUS_WORD::*;
+
+        let status = pmbus_read!(self.device, STATUS_WORD)?;
+        Ok(status.get_power_good_status() == Some(PowerGoodStatus::PowerGood))
+    }
+
+    pub fn firmware_revision(&self) -> Result<Mwocp68FirmwareRev, Error> {
```

Comment:
if i understand correctly, this function returns the revision of the primary MCU only (`XXXX` in the format string)? perhaps that should be documented here?